jscomp/frontend/bs_ast_invariant.ml

```diff
@@ -27,12 +27,11 @@
     it may fail third party ppxes
 *)
 let is_bs_attribute txt =
```

Comment:
This check was not working correctly for the non-bs attributes, see #6636.

Some more investigation needed though as adding "as" causes tests to fail. This should be done in a separate PR.